public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -305,21 +305,9 @@ export function getStreamSelectorsFromQuery(query: string): string[] {
 }
 
 export function requestSupportsPartitioning(allQueries: LokiQuery[]) {
-  const queries = allQueries.filter((query) => !query.hide);
-  /*
-   * For now, we only split if there is a single query.
-   * - we do not split for zero queries
-   * - we do not split for multiple queries
-   */
-  if (queries.length !== 1) {
-    return false;
-  }
-
-  if (queries[0].refId.includes('do-not-chunk')) {
-    return false;
-  }
+  const queries = allQueries.filter((query) => !query.hide).filter((query) => !query.refId.includes('do-not-chunk'));
```

Comment:
in main-branch there's an additional `IF` now here for loki-instant-queries. i think we should add it to here too WDYT?

Reply:
When I update this with main, I've been thinking that we could add instant queries as a third group to run, and make its interval the oringial one, unchunked. What do you think? Maybe it's a candidate for a dedicated PR.

Reply:
Synced with main.

Reply:
yes, that should work, and i agree that a separate PR will be best for that 👍